src/array/Vetor.java

```diff
@@ -39,7 +46,7 @@ public void adiciona(Aluno aluno) {
          - É necessario pegar a posição que se quer  inserir e mover as posições ocupadas para a direita
          - a complexidade aqui é O(n) pois tbm depende da quantidade de elementos no array
      */
-    public void adiciona(int posicao, Aluno aluno) {
+    void adiciona(int posicao, Aluno aluno) {
```

Comment:
Acho a retirada do modificador `public` desnecessária.  E se ele planejou a classe para ser utilizada em outras situações?

Reply:
Boa... mas até o momento não se fez necessário, quando tiver a utilidade de usar em algum outro lugar que seja fora do escopo do pacote, ai sim seria uma ótima oportunidade.  Posso estar errado, mas acredito que a partir do momento que ele colocar esse método público, ele deverá dar suporte para esse método para o resto da vida, pode ser a intenção, mas até o momento não foi demonstrado essa motivação. O que você acha? Lembro de termos falado algo sobre isso. Algo referente ao conteúdo desse artigo do Tio Martin:
```
Let’s assume I’m writing an application in a modern modular
language—to make things more
concrete, let’s assume this language is Java. My application thus
consists of several classes (and interfaces), each of which has a public interface. This public interface
of a class defines a group of methods that any
other class in the system can invoke.
While I’m enhancing a public method, I realize that one of its parameters is redundant—
I don’t need to use it in the method (maybe I
can get that value through some other route
or maybe I just don’t need it anymore). At this
point, I can eliminate that value from the
method signature, clarifying the method and
potentially saving work for its callers. Because
this method is public, any class in the system
can call it. Should I remove the parameter?
```
https://martinfowler.com/ieeeSoftware/published.pdf